examples/pump_events.rs

```diff
@@ -22,7 +22,7 @@ fn main() -> std::process::ExitCode {
     }
 
     impl ApplicationHandler for PumpDemo {
-        fn resumed(&mut self, event_loop: &ActiveEventLoop) {
+        fn can_create_surfaces(&mut self, event_loop: &ActiveEventLoop) {
             let window_attributes = Window::default_attributes().with_title("A fantastic window!");
             self.window = Some(event_loop.create_window(window_attributes).unwrap());
```

Comment:
Technically window creation is something that happens _before_ a surface can be created; this changed API just represents the async nature of it (and IIRC @kchibisov said it works like that on Wayland as well). Perhaps our examples shouldn't demonstrate creating window(s) here.

(Especially if we figure out how to make `winit::Window <=> android::Activity`, where there could be multiple `Activity`s contemporarily)

Reply:
Good point, I will address this in the follow-up PR.

Reply:
@daxpedda did you ever make a followup for this?

Reply:
I don't believe so.